### DISCUSSION TITLE
Copy values from Company/Departments/Company Details

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17198] [How to] [answered]
Hello,
Any way to get Company/Departments/Company Details values in a pdf template?

For example (Company name, NIF / CIF, Main email etc...) and from which module could these values be obtained?

Generating engine YetiFore PDF
YetiForce CRM 6.4 Service Pack v6

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://gitdeveloper.yetiforce.com/index.php?module=PDF&parent=Settings&view=Edit&record=33

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10198654/a2804940-f643-4215-a1ff-afe75af10f43)

https://gitdeveloper.yetiforce.com/index.php?module=FInvoice&view=Detail&record=579&parent=71&mid=129

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10198654/55c85e59-c821-4a00-ac02-0a99e5788f53)
